config.toml.in

```diff
@@ -8,7 +8,8 @@
 
 
 ## Storage
-# The storage section controls the where the safe file is stored.
+# The storage section controls where the safe is stored and which storage type
```

Comment:
This should be more generic. https://github.com/bndw/pick/pull/68 for example adds another option to `[storage]`